images/win/scripts/Tests/Validate-MongoDB.ps1

```diff
@@ -0,0 +1,27 @@
+################################################################################
+##  File:  Validate-MongoDB.ps1
+##  Desc:  Validate MongoDB
+################################################################################
+Describe "MongoDB" {
+
+    It "mongod is on path." {
+        if (Get-Command -Name 'mongod') {
+            Write-Host 'mongod is on path'
+        }
+        else {
+            Write-Host 'mongod not on path'
```

Comment:
We should not to print anything in Pester tests. Pester produces enough logging by itself

Reply:
Also we should not write `exit 1`. See my message above.
I suggest to rework test to 
```
It "mongod" {
   Validate-ZeroExitCode "mongod --version" | Should -Be 0
}
```

Reply:
The same for all other tests